Softeq.XToolkit.WhiteLabel.Forms/Controls/CustomTabbedPage.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Xamarin.Forms;
+
+namespace Softeq.XToolkit.WhiteLabel.Forms.Controls
+{
+    public class CustomTabbedPage : TabbedPage
```

Comment:
`Custom` isn't a good prefix for the library. Used in applications.

You cant use something like `TappedPageBase` (`-Base` is postfix style used in XToolkit (ViewModelBase, ActvityBase and etc.))